hardhat.config.ts

```diff
@@ -1,51 +1,107 @@
-import * as dotenv from "dotenv";
+import * as dotenv from "dotenv"
```

Comment:
You know, I actually _like_ the semi-colons ;)

Reply:
haha, really? i'll have to know why at some point